src/esp/core/esp.h

```diff
@@ -122,6 +122,24 @@ inline std::ostream& operator<<(std::ostream& os, const box3f& bbox) {
     return std::make_unique<__VA_ARGS__>(std::forward<Targs>(args)...); \
   }
 
+/**
```

Comment:
Is this really the best way to do this?

Reply:
I've seen two refs (SO and elsewhere) using this process to accomplish the desired result. If there's a better way, I'm hoping you guys can point it out so I can improve it. :)